src/ansys_sphinx_theme/__init__.py

```diff
@@ -2,6 +2,8 @@
 import os
 from pathlib import Path
 
+from .latex import generate_404  # noqa: F401
```

Comment:
```suggestion
from ansys_sphinx_theme.latex import generate_404  # noqa: F401
```
Though useful, I'd avoid whenever possible relative imports... Clarity prevails over "coding fast"